lib/read_geqdsk.pro

```diff
@@ -190,9 +192,25 @@ FUNCTION read_geqdsk,filename,grid,flux=flux,g=g
   
     flux = fluxgrid
     mask = replicate(1,grid.nr,grid.nz)
+    
+    for iz=0L,grid.nz-1 do begin
+        dbr_dr[*,iz]=Deriv(grid.r,reform(b_r[*,iz]))
+	dbt_dr[*,iz]=Deriv(grid.r,reform(b_t[*,iz]))
+	dbz_dr[*,iz]=Deriv(grid.r,reform(b_z[*,iz]))
+    endfor
+     
+    for ir=0L,grid.nr-1 do begin
+        dbr_dz[ir,*]=Deriv(grid.z,reform(b_r[ir,*]))
+	dbt_dz[ir,*]=Deriv(grid.z,reform(b_t[ir,*]))
+	dbz_dz[ir,*]=Deriv(grid.z,reform(b_z[ir,*]))
+    endfor
 
     equil={time:time,data_source:file_expand_path(filename), mask:mask, $
-           br:b_r,bt:b_t,bz:b_z,er:e_r,et:e_t,ez:e_z}
+           br:b_r,bt:b_t,bz:b_z,er:e_r,et:e_t,ez:e_z,$
+	   dbr_dr:dbr_dr,dbr_dz:dbr_dz,$
+	   dbt_dr:dbt_dr,dbt_dz:dbt_dz,$
+	   dbz_dr:dbz_dr,dbz_dz:dbz_dz }
+  
```

Comment:
1. The code blocks are not aligned properly.
2. This structure will be written to the equilibrium HDF5 file. Each of these new variables also need to have the `description` and `units`  defined in the [`write_equilibrium.pro`](https://github.com/D3DEnergetic/FIDASIM/blob/master/lib/write_equilibrium.pro) like its done [here](https://github.com/D3DEnergetic/FIDASIM/blob/master/lib/write_equilibrium.pro#L171)
3. This change would break backwards compatibility.